scripts/zones/Pashhow_Marshlands_[S]/mobs/Nommo.lua

```diff
@@ -1,9 +1,20 @@
 -----------------------------------
 -- Area: Pashhow Marshlands [S]
 --   NM: Nommo
+-- https://www.bg-wiki.com/ffxi/Nommo
 -----------------------------------
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    mob:setMod(xi.mod.PETRIFYRES, 10000)
```

Comment:
is it supposed to be super resistant or immune? if immune, edit mob pool, if resistant, this is set greatly higher than usual. unless the scale was change din another pr that escaped my notice this would be well beyond 100%

Reply:
There is no petrify immunity defined in core, unfortunately

Reply:
And you are right, this is a raw percent increase in resistance so 100 is what we want

Reply:
if he were immune I'd say add it to the enum, but looks like he's not. that looks like a trait proc. which means bg is wrong when they say immune to petrify.

Reply:
That screenshot is from my local after changing to 100.

I'm pretty confident he is literally immune, but I'll verify when I confirm the speed

Reply:
if on retail it says "completely resists the <whatever>" he truly immune

Reply:
Confirmed he's truly immune

Reply:
#5526

merry xmas in april

Reply:
Updated to use the immunity value